azure/v12.1.1/announcement.md

```diff
@@ -0,0 +1,3 @@
+**Giant Swarm release v12.1.1 for Azure is available**. This patch release fixes a problem causing the accidental deletion and reinstallation of Preinstalled Apps (such as CoreDNS) in 9.x.x tenant clusters. Please upgrade all older clusters to this version in order to prevent possible downtime. 
```

Comment:
```suggestion
**Giant Swarm release v12.1.1 for Azure is available**. This patch release fixes a problem causing the accidental deletion and reinstallation of Preinstalled Apps (such as CoreDNS). Please upgrade all older clusters to this version in order to prevent possible downtime. 
```

I think we just need to remove the 9.x.x. For Azure only the alpha uses node pools.